app/js/actions/draftLabOrderAction.js

```diff
@@ -0,0 +1,33 @@
+import {
+  ADD_DRAFT_LAB_ORDER,
+  DELETE_DRAFT_LAB_ORDER,
+  // DELETE_ALL_DRAFT_LAB_ORDERS_SUCCESS, // TODO: Delete action type
```

Comment:
Did you take a look at this? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotcommentoutcode